capstone/capapi/admin.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib import admin
+from . import models
+
+
+class APIUserAdmin(admin.ModelAdmin):
+    readonly_fields = ('date_joined', 'api_key')
+    list_display = ('email', 'last_name', 'first_name', 'api_key')
+
+    def api_key(self, instance):
+        return instance.get_api_key()
+    api_key.short_description = "API Key"
+    fields = (
```

Comment:
Nitpick: it's a little confusing to have `fields` down below the functions -- typically would go up with the other attributes at the top of the class.